lib/foreman_tasks_core/callback_action.rb

```diff
@@ -0,0 +1,11 @@
+module ForemanTasksCore
+  class CallbackAction < ::Dynflow::Action
+    def plan(callback, data)
+      plan_self(:callback => callback, :data => data)
+    end
+
+    def run
+      SmartProxyDynflowCore::Callback::Request.send_to_foreman_tasks(input[:callback], input[:data])
```

Comment:
Updated, we still reference it in SharableAction, but there is not other easy way how to do that. Anyway, no reason for this class to be here